config/locales/nl_input_elements.yml

```diff
@@ -144,9 +144,9 @@ nl:
     households_number_of_inhabitants_chart: "Aantal inwoners"
     households_inhabitants_per_household: "Aantal inwoners per huishouden"
     households_replacement_of_existing_houses: "Vervanging bestaande huizen"
-    households_space_heater_heatpump_air_water_electricity_share: "Warmtepomp (lucht)"
-    households_water_heater_heatpump_air_water_electricity_share: "Warmtepomp (lucht)"
-    households_water_heater_heatpump_ground_water_electricity_share: "Warmtepomp (grond)"
+    households_space_heater_heatpump_air_water_electricity_share: "Luchtwarmtepomp"
+    households_water_heater_heatpump_air_water_electricity_share: "Luchtwarmtepomp"
+    households_water_heater_heatpump_ground_water_electricity_share: "Warmtepomp (bodem)"
```

Comment:
@AlexanderWirtz should we not change this to 'Bodemwarmtepomp' for consistency with the 'Luchtwarmtepomp'?